test/vector-tile.test.js

```diff
@@ -2323,10 +2323,10 @@ describe('mapnik.VectorTile ', function() {
                 var validityReport3 = vtile.reportGeometryValidity({lat_lon:true});
                 var validityReport4 = vtile.reportGeometryValidity({web_merc:true});
                 assert.equal(simplicityReport.length, 0);
-                assert.equal(validityReport.length, 21); // Dataset not expected to be OGC valid
-                assert.equal(validityReport2.length, 14); // Dataset not expected to be OGC valid
-                assert.equal(validityReport3.length, 21); // Dataset not expected to be OGC valid
-                assert.equal(validityReport4.length, 20); // Dataset not expected to be OGC valid
+                assert.equal(validityReport.length, 25); // Dataset not expected to be OGC valid
+                assert.equal(validityReport2.length, 0); // Dataset not expected to be OGC valid
+                assert.equal(validityReport3.length, 26); // Dataset not expected to be OGC valid
+                assert.equal(validityReport4.length, 25); // Dataset not expected to be OGC valid
```

Comment:
@flippmoke can you offer an explanation for why these results are so different between clipper and wagyu?